install

```diff
@@ -137,26 +142,32 @@ do_install()
   pushd src
     # Avoid downloading opencv3; we already installed it from homebrew.
     wstool merge file://$(pwd)/../${WS}.rosinstall
-    wstool remove opencv3
+    #wstool remove opencv3
     wstool update -j8
 
-    # See: https://github.com/ros/catkin/pull/784
-    if [ -d catkin ]; then
-      curl https://raw.githubusercontent.com/ros/catkin/8a47f4eceb4954beb4a5b38b50793d0bbe2c96cf/cmake/catkinConfig.cmake.in > catkin/cmake/catkinConfig.cmake.in
+    if [ -d bond_core ]; then
+      curl curl https://raw.githubusercontent.com/ros/bond_core/1240bce50ca1538cdad07dc472d31797257e0851/bondcpp/src/bond.cpp > bond_core/bondcpp/src/bond.cpp
     fi
 
-    # Pending release: https://github.com/ros-visualization/rviz/pull/1165
-    if [ -d rviz ]; then
-      curl https://raw.githubusercontent.com/mikepurvis/rviz/21eac2bcc061bb623fd17aa449f6215c493b27f2/CMakeLists.txt > rviz/CMakeLists.txt
+    # https://github.com/ros-perception/vision_opencv/pull/225
+    if [ -d vision_opencv ]; then
+      curl https://raw.githubusercontent.com/meyerj/vision_opencv/9a9ad348abdfed010c70604cd964786f22220b30/cv_bridge/CMakeLists.txt > vision_opencv/cv_bridge/CMakeLists.txt
     fi
 
-    if [ -d image_pipeline ]; then
-      # Pending merge and release: https://github.com/ros-perception/image_pipeline/pull/303
-      curl https://raw.githubusercontent.com/mikepurvis/image_pipeline/d308d00aed5b66961f9e13ab7a50660a24be7c7f/image_proc/CMakeLists.txt > image_pipeline/image_proc/CMakeLists.txt
-      curl https://raw.githubusercontent.com/mikepurvis/image_pipeline/d308d00aed5b66961f9e13ab7a50660a24be7c7f/stereo_image_proc/CMakeLists.txt > image_pipeline/stereo_image_proc/CMakeLists.txt
+    # https://github.com/ros-perception/image_common/pull/82
+    if [ -d image_common ]; then
+      curl https://raw.githubusercontent.com/meyerj/image_common/51be08ba5dc352597a2ad5baabb7e92c6baa2634/camera_calibration_parsers/CMakeLists.txt > image_common/camera_calibration_parsers/CMakeLists.txt
+    fi
```

Comment:
This change looks super-gross. Basically any ROS package which wants to link to Python is going to have to do this dance on macOS. I'm fine to go this route for now just to get things moving along, but I wonder if perhaps a more transparent way might be to prepend CMAKE_PREFIX_PATH within ros-install-osx and have a small suite of wrapper CMake modules that provide the necessary indirection.

Thoughts?

Reply:
I noticed that my pull request links are wrong. @meyerj has a better idea on how to address this, which is independent of the OS.